docs/source/configuration/health-checks.mdx

```diff
@@ -5,11 +5,94 @@ description: Determining the router's status
 
 Health checks are often used by load balancers to determine whether a server is available and ready to start serving traffic.
 
-The Apollo Router supports a basic HTTP-level health check. This is enabled by default and is served at the URL path `/health`. This returns a `200` status code if the HTTP server is successfully serving. It does not invoke any GraphQL execution machinery.
-You can change this path by setting `health-check`:
-```yaml title="router.yaml"
-health-check:
-  listen: 127.0.0.1:9494
-  path: /health
-  enabled: true
+Because GraphQL queries can be sent as `GET` requests, a GraphQL query acts as an excellent health check. We recommend sending a simple query as a `GET` request with an appropriate `Content-type: application/json` header:
+
+```
+/?query={__typename}
+```
+
+You can optionally use a different query string that includes fields from one or more of your subgraphs. This has the added benefit of exercising the router's entire request lifecycle. Make sure to include the `Content-type: application/json` header in your request to meet the requirements for [CSRF protection](./csrf/).
```

Comment:
> This has the added benefit of exercising the router's entire request lifecycle.

Does running this specific query trigger execution ? :thinking_face:

Reply:
It does when doing the suggestion in the previous sentence:

> You can optionally use a different query string that includes fields from one or more of your subgraphs. 

Put another way, if you use a query for your health check that makes calls to fields that are served by subgraphs, then yes, the entire request life-cycle is exercised.